src/core/panel/plugin-panel.tsx

```diff
@@ -49,7 +49,7 @@ export abstract class UIPPluginPanel extends UIPPlugin {
     const hasToolbar = this.$toolbar?.children.length;
     return (
       <div className={type.is + '-header uip-plugin-header' + (this.label ? '' : ' no-label') + (hasToolbar ? ' has-toolbar' : '')}>
-        {this.$icon ? <span className='uip-plugin-header-icon' title={this.label}>{this.$icon}</span> : ''}
```

Comment:
Why does the `this.$icon` disappear ?
It is a used option, for example editor use this to display its icon
![image](https://github.com/user-attachments/assets/b189f6d4-d1f8-4fe9-9289-3bd6b04e8b76)